miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -511,7 +517,7 @@ private BulkPropagateSampleBackend(SampleClass targetSampleClass, BoxDto newBox,
     }
 
     @Override
-    protected SampleDto createDtoFromParent(Sample item) {
+    protected SampleDto createDtoFromParent(Sample item) { // TODO: identityConsentLevel
```

Comment:
Is this TODO still needed? Looks like it's implemented below in 530-532.

Reply:
removed